tests/FakeItEasy.Tests.TestHelpers/UsingCultureAttribute.cs

```diff
@@ -23,28 +23,16 @@ public UsingCultureAttribute(string cultureName)
 
         public override void After(MethodInfo methodUnderTest)
         {
-#if FEATURE_THREAD_CURRENTCULTURE
             Thread.CurrentThread.CurrentCulture = this.originalCulture;
             Thread.CurrentThread.CurrentUICulture = this.originalUiCulture;
-#else
-            CultureInfo.CurrentCulture = this.originalCulture;
-            CultureInfo.CurrentUICulture = this.originalUiCulture;
-#endif

```

Comment:
Any specific reason to keep `Thread.CurrentThread.CurrentCulture` instead of `CultureInfo.CurrentCulture`? Both work, but `CultureInfo.CurrentCulture` has always felt more natural to me.
Doesn't really matter though; if you prefer to keep it this way, I'll merge as is.

Reply:
Swapped!